src/assets/stylesheets/custom.scss

```diff
@@ -138,8 +136,39 @@ $textCenter: center;
 .db{ display: block; }
 .textCenter{ text-align: $textCenter; }
 .pointer{ cursor: pointer; }
+//patternfly-site.css
+h2{
+  font-size: em(18);
+  font-weight: 500;
+  line-height: 1.4;
+}
+//card-view
 
-//user -gravatar
+.card-pf-view {
+  .card-pf-top-element {
+    .card-pf-icon-circle {
+      @include Border(em(0.8), solid, $color-pf-blue-300); //#39a5dc;
+      @include borderRadius(50%);
+      @extend .db;
+      font-size: em(36);
+      height: em(34); //2.84em
```

Comment:
We should bounce it back against @fabric8io/uxd team if they'd prefer a static `px` value instead of `em` here. From what I understand, serving it with different form factor isn't a choice here. Do we need to force `em`? Although, same can be asked about why 102px has been prescribed to be precise, but well...

(To clarify, I'm grossly fine with the current implementation, it's not hindering usability IMO in any way, but I don't like the inconsistency with the UXD spec vs implementation. Either we need to fix/justify the spec, or follow it properly.)